ui/src/user/components/__tests__/SignUpPage.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SignUpPage from '../SignUpPage';
+
+jest.mock('../../../actions/user');
+
+describe('SignUpPage', () => {
+  it('renders page', () => {
+    const wrapper = shallow(
+      <SignUpPage loading={false} onSubmit={jest.fn()} />
+    );
```

Comment:
needs `dive()` to assert form fields, as you can see in the snapshot they are not there.